### PR TITLE
fix: Fix issues with inputs on iOS

### PR DESCRIPTION
**Pull request by alimpens**

### CODE REVIEW

File: packages/css/src/components/select/select.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
  */
 
 @mixin reset {
-  appearance: none;
+  appearance: none; // Reset native appearance to hide default chevron
```
Comment (RubenSibon) [inline packages/css/src/components/select/select.scss:7]: Why not a webkit prefix here?

Comment (RubenSibon) [reply packages/css/src/components/select/select.scss:7]: I guess because it's not an input. Yeah.

**Review by RubenSibon — APPROVED** [review]
LGTM, I assume this has been thoroughly tested on all supported devices and browsers.

**Comment by alimpens** [comment]
> LGTM, I assume this has been thoroughly tested on all supported devices and browsers.

Thanks! I tested it on Chrome, Firefox and Edge on Windows, Chrome on Android and Safari 15.2, 15.6 and 16.6 on iOS.

File: packages/css/src/components/select/select.scss (hunk 1 above)
Comment (alimpens) [reply packages/css/src/components/select/select.scss:7]: `appearance: none` is used here to hide the default chevron. That isn't necessary for Safari < 15.4 (which are the browsers for which we sometimes need the prefix)

**Review by VincentSmedinga — APPROVED** [review]

**Pull request merged**